schema/JF_TRAN_PAC_MEMO.json

```diff
@@ -348,7 +351,7 @@
       "type": ["string", "null"],
       "minLength": 0,
       "maxLength": 9,
-      "pattern": "^[ -~]{0,9}$",
+      "pattern": "^C[0123456789]{0,9}$",
```

Comment:
This can be changed to ^C[0-9]{0,9}$ which is more succinct. The same pattern can be applied to the FILER_COMMITTEE_ID_NUMBER

Reply:
Feedback incorporated